pyqt-apps/siriushla/as_ps_cycle/pvnames_tree.py

```diff
@@ -0,0 +1,228 @@
+"""Magnet selection tree view."""
+import re
+from collections import namedtuple
+import threading
+
+from qtpy.QtCore import Qt, QSize, Signal, QThread
+from qtpy.QtWidgets import QTreeWidget, QTreeWidgetItem, QProgressDialog
+
+from siriuspy.search.ma_search import MASearch
+from siriuspy.namesys import SiriusPVName
+
+
+class PVNameTree(QTreeWidget):
```

Comment:
@gu1lhermelp, this is a very nice Widget and I think it could be moved to the widgets package.

Reply:
Moved